slo-workload/src/DataGenerator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace YdbPlatform\Ydb\Slo;
+
+use YdbPlatform\Ydb\Session;
+use YdbPlatform\Ydb\Table;
+use YdbPlatform\Ydb\Types\DoubleType;
+use YdbPlatform\Ydb\Types\TimestampType;
+use YdbPlatform\Ydb\Types\Uint64Type;
+use YdbPlatform\Ydb\Types\Utf8Type;
+
+class DataGenerator
```

Comment:
Do you use DataGenerator as namespace?

What about use this as normal class with state and use instance of the class for generate data?

Global state is bad idea at most cases.